content/docs/reference/reference.json

```diff
@@ -925,7 +925,7 @@
     "title": "Pass Identity Headers (per route)",
     "description": "When set, Pass Identity Headers passes identity headers to the upstream application.",
     "short_description": "Pass identity headers to the upstream application",
-    "path": "/routes/pass-identity-headers-per-route",
```

Comment:
It I'd a 404

Reply:
The new global- and route-level URLs are on `main`, or `vNext` on the docs site. This changes the URL to `main.docs.pomerium.com/docs` until we cut v0.25. Just commenting this here in case we need to reference this in the future. Closing this PR for now.